CMakeLists.txt

```diff
@@ -63,6 +67,7 @@ endif()
 
 #===== Local modules =====
 list (APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+list (APPEND CMAKE_MODULE_PATH "/glade/work/haiyingx/miniconda3/envs/z5-py36/share/cmake-3.14/Modules/")
```

Comment:
This is not okay for a general port - can you move these modules to ./cmake/

Reply:
I need to test if I move them.

Reply:
Removed